### PR TITLE
Implemented Zhao (2020) close #93

### LINKED ISSUE #93: add Zhao (2020)
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095099/pdf/40484_2020_Article_199.pdf

### PR DESCRIPTION
**Pull request by IvaEwert**
I have the same issue as this one: https://github.com/PEtab-dev/PEtab/issues/471. 
Parameter bounds for the parameter R were specified by myself.

### CODE REVIEW

**Comment by LeonardSchmiester** [comment]
> I have the same issue as this one: [PEtab-dev/PEtab#471](https://github.com/PEtab-dev/PEtab/issues/471).

This is only related to plotting, but not to the implementation of the model, right?

**Comment by LeonardSchmiester** [comment]
The tests currently require the yaml file to have the same name as the folder. Could you rename one of the yaml files to `Zhao_QuantBiol2020.yaml`?

**Comment by IvaEwert** [comment]
> 
> 
> > I have the same issue as this one: [PEtab-dev/PEtab#471](https://github.com/PEtab-dev/PEtab/issues/471).
> 
> This is only related to plotting, but not to the implementation of the model, right?

Yes

File: Benchmark-Models/Zhao_QuantBiol2020/simulatedData_Zhao_QuantBiol2020.tsv [hunk 1]
```diff
@@ -0,0 +1,170 @@
+	observableId	preequilibrationConditionId	simulationConditionId	simulation	time	observableParameters	noiseParameters	datasetId
+0	observable_confirmed_infected		model1_China_Stage_I	3445.6360071261174	30		sd_confirmed_infected_China_Stage_I	model1_data_observable_confirmed_infected_China_Stage_I
```
Comment (LeonardSchmiester) [inline Benchmark-Models/Zhao_QuantBiol2020/simulatedData_Zhao_QuantBiol2020.tsv:2]: Could you remove the index column here?

**Comment by LeonardSchmiester** [comment]
This one can be closed then, right?!

**Pull request closed without merging**